source/help/messaging/attaching-files.rst

```diff
@@ -1,64 +1,50 @@
-Attaching Files
+Sharing Files
 ===============
 
---------------
+With attachments, you can share additional information that will help your 
+team to visually understand your ideas. Sharing videos, voice recordings,
+screenshots, and photos will make your messages more effective and clear.
 
-Attachment Methods
+How to Attach Files
 ------------------
-Attach a file by drag and drop or by clicking the attachment icon in the
-message input box.
-
-Drag and Drop
-~~~~~~~~~~~~~
-Upload a file or selection of files by dragging the files from your
-computer into the right-hand sidebar or center pane. Dragging and
-dropping attaches the files to the message input box. After dropping files, can
-optionally type a message and press **ENTER** to post.
-
-Attachment Icon
-~~~~~~~~~~~~~~~
-Alternatively, upload files by clicking the grey paperclip icon inside
-the message input box. This opens up your system file viewer where you
-can navigate to the desired files and then click **Open** to upload the
-files to the message input box. Optionally type a message and then press
-**ENTER** to post.
-
-Pasting Images
-~~~~~~~~~~~~~~
-On Chrome and Edge browsers, it is also possible to upload files by
-pasting them from the clipboard. This is not yet supported on other
-browsers.
+You can attach files in the following ways:
+
+-  **Use the attachment icon** - click the paperclip icon inside the message input box
+-  **Drag and Drop**
+-  **Paste from the clipboard** - this feature is supported in Chrome and Edge browsers
 
 File Previewer
 --------------
-Mattermost has a built in file previewer that is used to view media,
-download files and share public links. Click the thumbnail of an
-attached file to open it in the file previewer.
+Mattermost has a built-in file previewer that you can use to:
 
-Sharing Public Links
-~~~~~~~~~~~~~~~~~~~~
-Public links allow you to share file attachments with people outside
-your Mattermost team. Open the file previewer by clicking on the
-thumbnail of an attachment, then click **Get Public Link**. This opens a
-dialog box with a link to copy. When the link is shared and opened by
-another user, the file will automatically download.
+-  Download files
+-  Share public links
+-  View media
 
-If **Get Public Link** is not visible in the file previewer and you
-prefer the feature enabled, you can request that your System Admin
-enable the feature from the System Console under **Security** > **Public
-Links**.
+Click the thumbnail of an attached file to open it in the file previewer.
 
 Downloading Files
 ~~~~~~~~~~~~~~~~~
-Download an attached file by clicking the download icon next to the file
-thumbnail or by opening the file previewer and clicking **Download**.
+You can download an attached file by:
+
+-  Clicking the download icon next to the file thumbnail
+
```

Comment:
A list should have two or more items. If only one item, then it's not a list. This should be:

`You can download an attached file by clicking the download icon next to the file thumbnail.`